paper_on_queues_and_stack_structures.md

```diff
@@ -0,0 +1,41 @@
+# Queues and Stack Structures  
```

Comment:
Adding the "Introduction to..." for this title seems to be more fitting in my eyes.

Reply:
Ok thanks, got it!